common/src/device/transport/mqtt/ClientWebSocketChannel.cs

```diff
@@ -316,34 +310,36 @@ public override int GetHashCode()
 
         public static bool operator <(ClientWebSocketChannel left, ClientWebSocketChannel right)
         {
-            return ReferenceEquals(left, null) ? !ReferenceEquals(right, null) : left.CompareTo(right) < 0;
+            return left is null ? right is object : left.CompareTo(right) < 0;
```

Comment:
The logic used for > looks easier to follow.
return left is object && left.CompareTo(right) < 0;